watch.go

```diff
@@ -180,7 +204,8 @@ func (fb *Firebase) Watch(notifications chan Event) error {
 
 				// TODO: handle
 			case "rules_debug":
-				log.Printf("Rules-Debug: %s\n", txt)
+				txt, _ := ioutil.ReadAll(resp.Body)
+				log.Printf("Rules-Debug: %s\n", string(txt))
```

Comment:
I don't know how the "rules_debug" is used. As far as I remember, your
original code gives the full body, including the "event:" line. In case of
malformed response, the eventData would be empty. So this is why I chose to
re-read the full body.


█  Jacques

On 31 July 2016 at 20:57, Steven Berlanga <notifications@github.com> wrote:

> In watch.go
> <https://github.com/zabawaba99/firego/pull/50#discussion_r72908920>:
>
> > @@ -180,7 +204,8 @@ func (fb *Firebase) Watch(notifications chan Event) error {
> >
> >  				// TODO: handle
> >  			case "rules_debug":
> > -				log.Printf("Rules-Debug: %s\n", txt)
> > +				txt, _ := ioutil.ReadAll(resp.Body)
> > +				log.Printf("Rules-Debug: %s\n", string(txt))
>
> Wouldn't this be the eventData?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zabawaba99/firego/pull/50/files/91637d68c23bcd6761930b23882b18621b759365#r72908920>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAB2iPTtex2wypPuhQlU-SfjUBIVJKpsks5qbPAZgaJpZM4JYHTS>
> .
>